src/todo_final.ts

```diff
@@ -32,48 +40,49 @@ function testItems() {
 function completeList() {
 	//filter iterate through each item of the array
 
-	let completedItems = todoItems.filter((item) => item.checked === true);
-	for (var i = 0; i < completedItems.length; i++) {
-		console.log(completedItems[i].id + " " + completedItems[i].text);
+	const completedItems = todoItems.filter((item) => item.checked === true);
+	for (const item of completedItems) {
+		console.log(item.id + " " + item.text);
 	}
 }
 
 //list uncompleted todos
 function uncompletedList() {
-	/// filter is an array method that returns the position of an element
-	// in the array
+	/*
+	 *filter is an array method that returns the position of an element in the array
```

Comment:
There should be a space between the `*` and the comment text, like `* filter is an array method that returns the position of an element in the array`